.gitbook.yaml

```diff
@@ -0,0 +1,10 @@
+redirects:
+  guidelines/certification: your-journey/certification.md
+  guidelines/communicate: events/communicate.md
+  use-cases/payment-automation/README: use-cases/payment-automation.md
```

Comment:
issue: This is not needed because the resulting URL will be the same.
```suggestion
```

Reply:
Will it? I assume the ".md" is needed here, otherwise it will default to "README.md", or am I mistaken?. So it should be: `use-cases/payment-automation: use-cases/payment-automation.md`

Reply:
I meant the key: there is no URL like `use-cases/payment-automation/README`, only `use-cases/payment-automation`. And that translates to the same URL as `use-cases/payment-automation.md`. Your proposal would result in a loop (which Gitbook would, hopefully, ignore).

Reply:
The gist is: the key is path as you see it in the URL on the final site. The value is path to the markdown file.